Core/include/Acts/EventData/ParameterConcept.hpp

```diff
@@ -64,16 +64,19 @@ namespace concept {
         identical_to<double, time_returntype_t, const P>;
     constexpr static bool surfacebound_covariance_exists =
         identical_to<const std::optional<BoundSymMatrix>&,
-                     covariance_returntype_t, const P>;
+                     covariance_returntype_t,
+                     const P>;
     constexpr static bool surfacebound_parameters_exists =
         identical_to<BoundVector, parameters_returntype_t, const P>;
 
     /// Tie all surface bound results together
-    constexpr static bool boundValue =
-        require<reference_surface_exists, surfacebound_position_exists,
-                surfacebound_momentum_exists, surfacebound_charge_exists,
-                surfacebound_time_exists, surfacebound_covariance_exists,
-                surfacebound_parameters_exists>;
+    constexpr static bool boundValue = require<reference_surface_exists,
+                                               surfacebound_position_exists,
+                                               surfacebound_momentum_exists,
+                                               surfacebound_charge_exists,
+                                               surfacebound_time_exists,
+                                               surfacebound_covariance_exists,
+                                               surfacebound_parameters_exists>;
```

Comment:
Just commenting so this shows on the PR as an example how it would look after the change